packages/graphql/src/translate/where/create-where-clause.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { PointField, PrimitiveField } from "../../types";
+import { RelationshipWhereOperator, WhereOperator } from "./types";
+
+const comparisonMap: Record<Exclude<WhereOperator, RelationshipWhereOperator>, string> = {
+    NOT: "=",
+    // Numerical
+    GT: ">",
+    GTE: ">=",
+    LT: "<",
+    LTE: "<=",
+    // Distance
+    DISTANCE: "=",
+    // String
+    NOT_CONTAINS: "CONTAINS",
+    CONTAINS: "CONTAINS",
+    NOT_STARTS_WITH: "STARTS WITH",
+    STARTS_WITH: "STARTS WITH",
+    NOT_ENDS_WITH: "ENDS WITH",
+    ENDS_WITH: "ENDS WITH",
+    // Regex
+    MATCHES: "=~",
+    // Array
+    NOT_IN: "IN",
+    IN: "IN",
+    NOT_INCLUDES: "IN",
+    INCLUDES: "IN",
+};
```

Comment:
This is now declared twice - here and also in `packages/graphql/src/translate/where/utils.ts`. Can you choose the one which is most appropriate?

Reply:
I'm really falling down here. Sorry. Fix in https://github.com/neo4j/graphql/pull/796/commits/f0118d1cf2e1bc0fc6c2fb2e99c3027e7c38fdee

Reply:
Not at all, this has been open for some time, it's easily done!

Let's get this merged. 🙂